projet/projects/urls.py

```diff
@@ -11,4 +11,20 @@
         views.ProjectDetailUpdateDelete.as_view(),
         name="update-delete",
     ),
+    path(
+        "accept/<int:project_id>/<int:contributor_id>/",
+        views.AcceptContributor.as_view(),
+        name="accept-contribution",
+    ),
+    path(
+        "contributors/<int:project_id>/",
+        views.get_projects_contributors,
+        name="contributors",
+    ),
+    path(
+        "contributors/<int:project_id>/<str:statum>/",
+        views.get_project_pending_contributors,
+        name="pending-accepted-contributors",
+    ),
```

Comment:
Depending on the statum argument when the project id is passed in and the statum is either pending or accepted the contributors for the project either pending or accepted will be returned.
Later if no issues will be caused the function handling this route will be merged to the all contributors function so that if no statum is passed all contributors will be returned or else the pending or accepted ones will be returned.